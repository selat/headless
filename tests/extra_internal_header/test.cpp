#include "external.hpp"
#include "totally_unused.hpp"

void test() {
  SuperType value = 10.0;
}
