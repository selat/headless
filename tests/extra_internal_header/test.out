Please add the following headers:
Please remove the following headers:
totally_unused.hpp
